github_tutorial.md

```diff
@@ -94,67 +97,83 @@ De beheerder van de standaard zal dus de volgende handelingen doen voor een vers
 	* Klik op "Propose new file"
 
 2. Ik wil een (tekst)document wijzigen
-	* Op en het document vanuit tab "Code" door te klikken op de bestandsnaam
+	* Ga naar de tab "Code"
+	* Kies daar de branch waarin je de wijzigingen wil aanbrengen
+	* Open het document dat je wil aanpassen door te klikken op de bestandsnaam
 	* Klik op het pen-icoon
-	* Voor de toevoegingen of wijzigingen door in de inhoud van het document
-	* Vul bij "Propose new file" een korte titel voor de toevoegingen of wijzigingen en een uitgebreide toelichting hiervan
-	* Klik op "Propose file change"
-
-3. Ik wil een opmerking maken over een bijdrage van iemand anders
+	* Voer de toevoegingen of wijzigingen door in de inhoud van het document
+	* Commit de wijzigingen in het document door op "Commit changes" te klikken waarbij je de radio button "Commit directly to the nnnnn branch." selecteert. "nnnnn" staat voor de branch waarin je deze wijziging wilde aanbrengen.
+
+3. Ik wil een "Pull request" indienen
+	* Open de tab "Pull requests"
+	* Klik op "New pull request"
+	* Selecteer bij "base fork" de repository en bij "base" de branch waar het pull request op moet worden geplaatst
+	* Selecteer bij "head fork" de repository en bij "compare" de branch waarin de wijziging is aangebracht
+	* Klik op "Create pull request" en ken er een naam aan toe die voldoet aan de volgende conventie:
+	  'fixit [nummer issue]'. Bij aanvaarden van de pull request wordt meteen het issue gesloten
+	* Scroll evt. naar beneden en klik op "Create pull request";
+	* Scroll naar beneden naar "Merge pull request";
+	* Kies een van de 3 opties
```

Comment:
Wanneer moet/wil je hier iets anders dan de eerste optie kiezen? Ik denk dat niemand die deze tutorial nodig heeft iets anders dan de eerste optie moet willen.

Reply:
Is aangepast.